File: PRAK604_2310817210022_Muhammad_Adh-Dhiya'Us_Salim.c
#include <stdio.h>
#include <string.h>

int main() {
    char KalimatSatu[1000], KalimatDua[1000];
    printf("Masukkan kalimat pertama: ");
    gets(KalimatSatu);
    printf("Masukkan kalimat kedua: ");
    gets(KalimatDua);

    if (strlen(KalimatSatu) != strlen(KalimatDua)) {
        printf("Panjang kalimat berbeda, pesan palsu\n");
    } else {
        int sama = 0, beda = 0;
        for (int i = 0; i < strlen(KalimatSatu); i++) {
            if (KalimatSatu[i] == KalimatDua[i] && KalimatSatu[i] != ' ' && KalimatDua[i] != ' ') {
                printf("*");
                sama++;
            } else if (KalimatSatu[i] == ' ' && KalimatDua[i] == ' ') {
                printf(" ");
            } else {
                printf("#");
                beda++;
            }
        }
        printf("\n");
        printf("* = %d\n", sama);
        printf("# = %d\n", beda);

        if (sama > beda) {
            printf("Pesan asli\n");
        } else {
            printf("Pesan palsu\n");
        }
    }

    return 0;
}

File: PRAK604_2310817210022_Muhammad_Adh-Dhiya'Us_Salim.py
KalimatSatu = input()
KalimatDua = input()

if len(KalimatSatu) != len(KalimatDua):
    print('Panjang kalimat berbeda, pesan palsu')
else:
    sama = 0
    beda = 0
    for i in range(len(KalimatSatu)):
        if KalimatSatu[i] == KalimatDua[i] and KalimatSatu[i] != " " and KalimatDua[i] != " ":
            print("*", end="")
            sama += 1
        elif KalimatSatu[i] == ' ' and KalimatDua[i] == ' ':
            print(" ", end="")
        else:
            print("#", end="")
            beda += 1
    print()
    print(f'* = {sama}')
    print(f'# = {beda}')
    if sama > beda:
        print('Pesan asli')
    else:
        print('Pesan palsu')